### PR TITLE
[16.0][MIG] coupon_limit > loyalty_limit: Migration to version 16.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
Renamed to loyalty_limit

cc @Tecnativa TT44319

@chienandalu @ernesto-garcia-tecnativa please review

### CODE REVIEW

**Review by chienandalu — COMMENTED** [review]
Please add a temporal reference to your dependency so we can se the whole picture: https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference(s)-to-another-pull-request(s)

**Comment by pilarvargas-tecnativa** [comment]
> Please add a temporal reference to your dependency so we can se the whole picture: https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference(s)-to-another-pull-request(s)

thanks :)

File: loyalty_limit/migrations/16.0.1.0.0/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        "loyalty_salesmen_limit",
+        "rule_max_salesman_application",
+        "max_salesman_application",
+    ),
+]
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if not openupgrade.table_exists(env.cr, "mass_editing_line"):
```
Comment (chienandalu) [inline loyalty_limit/migrations/16.0.1.0.0/pre-migration.py]: What's this?

**Review by chienandalu — COMMENTED** [review]
Looking good. Some doubts though:

File: loyalty_limit/migrations/16.0.1.0.0/pre-migration.py (hunk 1 above)
Comment (pilarvargas-tecnativa) [reply loyalty_limit/migrations/16.0.1.0.0/pre-migration.py]: It has slipped

**Comment by pilarvargas-tecnativa** [comment]
@CarlosRoca13 take a look please

**Review by CarlosRoca13 — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]
Let's go :)

**Comment by pilarvargas-tecnativa** [comment]
ping @pedrobaeza :D

**Pull request merged**